index.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
     "object-curly-newline": 1,
     "object-property-newline": ["error", { "allowAllPropertiesOnSameLine": true }],
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "react/jsx-fragments": [1, "element"],
```

Comment:
If we want to enforce it consistently/strictly I think we'll want to make it an error with 2 instead of a warning with 1.

Reply:
good point @mjaltamirano!